RevisionHistory.html

```diff
@@ -10,6 +10,20 @@
     <h1>DFTFringe Version History</h1>
     <ul>
 
+        <li>Version 8.2.0</li>
+        <ul>
+            <li>Gaussian blur no longer turns edges (and other artifacts) -- zernike blur no longer needed for foucault/ronchi/star test simulations</li>
+            <li>Profile plot can maximize once again along with ability to change units among inches, mm, percent.</li>
+            <li>Zonal Percent correction graph from 7.3.1 moved to the context menu (right click) of the profile plot</li>
+            <li>darkmode disabled (was a problem for people with win11/win12 who set windows to dark mode)</li>
+            <li>annulus feature help text improved</li>
```

Comment:
```suggestion
            <li>Darkmode disabled (was a problem for people with win11/win12 who set windows to dark mode)</li>
            <li>Annulus feature help text improved</li>
```

Win 12 ?? did you mean 10/11 ?